src/common/Pic/__tests__/Pic.test.js

```diff
@@ -0,0 +1,377 @@
+import React from 'react';
```

Comment:
Do you think you could rebase the commit to do a `git mv test/component.js src/common/Pic/__tests__/Pic.test.js` so that we keep the previous history since most of that file is in here now.

Reply:
I don't mine doing this however I'm not very versed in doing rebase's so I'm not sure I would feel comfortable heading down this path. I can give it a go though and come back to you if it does work. 

Any quick suggestions

Reply:
If you rebase to the commit before the one where this file was created and the other was removed and make a separate commit for the git mv to here. Git should recognize it as a moved file.